magento/inventory
# MSI-1667: 'Salable Quantity' column is empty after deploying small.xml preset.

### Description


### Fixed Issues (if relevant)

1. magento-engcom/msi#1667: 'Salable Quantity' column is empty after deploying small.xml preset.

### Manual testing scenarios

1. Execute:
```
bin/magento setup:perf:generate-fixtures setup/performance-toolkit/profiles/ce/small.xml
``` 
2. Navigate to Catalog > Products
3. Products Salable Quantity should be equal to 100500

### Contribution checklist
